lms/templates/learner_dashboard/programs.html

```diff
@@ -0,0 +1,31 @@
+<%page expression_filter="h"/>
+<%inherit file="../main.html" />
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.template import RequestContext
+import third_party_auth
+from third_party_auth import pipeline
+from django.core.urlresolvers import reverse
+import json
```

Comment:
You can get rid of these imports. You should only need the js_utils imported below.